site/en/guides/border-router/espressif-esp32.md

```diff
@@ -1,69 +1,73 @@
 # ESP Thread Border Router
 
-Contributor: https://github.com/gjc13
+Contributor: https://github.com/gjc13 https://github.com/zwx1995esp
```

Comment:
Can you add yours on a separate line.  So there will be two individual contributor lines:

Contributor: https://github.com/gjc13
Contributor: https://github.com/zwx1995esp

Also, can you confirm you're affiliated with Espressif?  (seems so from your profile)  That way we can ensure the byline uses the Espressif logo on openthread.io.

Reply:
Yes, I am working for the Espressif.